datasets/RES11_nl2019/RES11_nl2019.derived.ad

```diff
@@ -0,0 +1,223 @@
+- area = RES11_nl2019
```

Comment:
This should be RES11_holland_rijnland, seems like something went wrong?

Reply:
I have fixed the typo. The correct data is now shown in the ETM, see the 'land use of solar and wind' figure.